src/applications/claims-status/tests/actions.unit.spec.js

```diff
@@ -294,7 +294,7 @@ describe('Actions', () => {
     it('should call dispatch and pollStatus', () => {
       const dispatchSpy = sinon.spy();
       const pollStatusSpy = sinon.spy();
-      getClaimsV2(pollStatusSpy)(dispatchSpy);
+      getClaimsV2({ poll: pollStatusSpy })(dispatchSpy);
```

Comment:
These unit tests are the only place we were overriding the default `getClaimsV2` `poll` argument. So I've updated them all to use the new options object.